src/components/CardModal/CardModal.js

```diff
@@ -39,6 +39,7 @@ const CardModal = ({ card }) => {
   const handleOnBlur = () => {
     setSelected(false);
     setEditableContent(card.content);
+    setRows(1);
```

Comment:
When we close the edit title textarea, we want to reset the rows as they'll be calculated again on focus. If we do not set this, the calculations in `handleOnFocus` below will be incorrect